app/README.md

```diff
@@ -0,0 +1,62 @@
+# Development Tips
+
+## Start the Rest API w/o security
```

Comment:
Why is this needed?  Like what reason would I need to go through these steps?  How is this different than the normal development workflow?

Reply:
@gashcrumb It allows for quick turn around times when working on an issue that spans REST and UI. That said we should add the instructions for the `normal` workflow too if you can.

Reply:
Yep, it's here -> https://github.com/syndesisio/syndesis/blob/master/app/ui/README.md#day-to-day-workflow